site.cfg.example

```diff
@@ -150,8 +150,10 @@
 # OpenBLAS (64-bit ILP64)
 # -----------------------
 # It is possible to also use OpenBLAS compiled with 64-bit integer
-# size (ILP64) but no symbol name changes. To do that, set the
-# environment variables NPY_USE_BLAS_ILP64=1,
+# size (ILP64) but no symbol name changes. This is the default in the wheels
+# provided by the project. You can force 32-bit compiled OpenBLAS by setting
```

Comment:
The openblas64_ actually comes before the suffixless libs:
https://github.com/numpy/numpy/blob/df9c973f28fb9c0e713db29285b07b57c622652a/numpy/distutils/system_info.py#L1902

openblas-libs also provides only the 64_ libs: https://anaconda.org/multibuild-wheels-staging/openblas-libs/files

This is on purpose, since using ILP64 Numpy on Linux inside a Python interpreter embedded in an application that is linked dynamically to 32-bit BLAS, will result to segfaults if the 64-bit symbol names have not been renamed.